crates/esplora/src/blocking_ext.rs

```diff
@@ -10,59 +10,108 @@ use bdk_chain::{
     BlockId, ConfirmationTimeHeightAnchor, TxGraph,
 };
 use bdk_chain::{Anchor, Indexed};
-use esplora_client::TxStatus;
+use esplora_client::{OutputStatus, Tx, TxStatus};
 
 use crate::anchor_from_status;
 
-/// [`esplora_client::Error`]
+/// [`esplora_client::Error`] wrapped in a [`Box`].
```

Comment:
you don't need to say it's wrapped in a `Box`. No one cares about this and it's right there in the type definition.

Reply:
we may not need to box the error anymore after rust-esplora-client moved to using minreq